test/docs.js

```diff
@@ -189,7 +191,7 @@ describe('documentation', function() {
 
           // in:  fs.anyMethod('--any-file-path--
           // out: fs.anymethod('--a-temporary-file--
-          .replace(/(fs\.[^(]+\(['"])([^'"]*)/g, '$1' + temporaryFile.name);
```

Comment:
While there is the manual unlink in the after hook, we know that's not always reliable from experience-- check how many buckets/clusters/instances/etc etc are in your GCP console that start with "GCLOUD-TEST" 😉. In general, the thought was since we have a module that's for this exact purpose, we shouldn't have to be writing it out ourselves.

> I tried your previous suggestion and it didn't work

I'm not sure if that means normalize, manual C:\ -> C:\\ replacement, or both. You also had an idea about using the async way of creating a temp file. But if you're done playing with the temp module, you can just merge the docs changes and I'll give it a go later this week.